Applications/Classification/jni/Android.mk

```diff
@@ -9,73 +9,32 @@ endif
 
 ifndef NNTRAINER_ROOT
 NNTRAINER_ROOT := $(LOCAL_PATH)/../../../libs/arm64-v8a
-NNTRAINER_INCLUDES := $(LOCAL_PATH)/../../../nntrainer/include/
+NNTRAINER_INCLUDES := $(LOCAL_PATH)/../../../nntrainer/include
 endif
 
-NNTRAINER_APPLICATION := $(LOCAL_PATH)/../../
+NNTRAINER_APPLICATION := $(LOCAL_PATH)/../..
 
 include $(CLEAR_VARS)
 
+TENSORFLOW_VERSION := 1.13.1
+
```

Comment:
Is it the version used by nnstreamer? I think we have to maintain the consistency with nnstreamer.

Reply:
yes,

Please refer to https://github.com/nnstreamer/nntrainer/issues/20#issuecomment-615063429, and https://launchpad.net/~nnstreamer/+archive/ubuntu/ppa